docs/remote_inference_blueprints/sagemaker_connector_blueprint.md

```diff
@@ -0,0 +1,454 @@
+### Sagemaker connector blueprint example for embedding:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "sagemaker: embedding",
+  "description": "Test connector for Sagemaker embedding model",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "credential": {
+    "access_key": "<PLEASE ADD YOUR AWS ACCESS KEY HERE>",
+    "secret_key": "<PLEASE ADD YOUR AWS SECRET KEY HERE>",
+    "session_token": "<PLEASE ADD YOUR AWS SECURITY TOKEN HERE>"
+  },
+  "parameters": {
+    "region": "<PLEASE ADD YOUR AWS REGION TOKEN HERE>",
+    "service_name": "sagemaker"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "headers": {
+        "content-type": "application/json"
+      },
+      "url": "https://runtime.sagemaker.${parameters.region}.amazonaws.com/endpoints/lmi-model-2023-06-24-01-35-32-275/invocations",
+      "request_body": "[\"${parameters.inputs}\"]"
+    }
+  ]
+}
+```
+
+#### Sample response
+```json
+{
+  "connector_id": "XU5UiokBpXT9icfOM0vt"
+}
+```
+
+### Corresponding Predict request example:
```

Comment:
As we can predict what's the model deployed to Sagemaker on user side, so we should remove the predict request example and sample response.